### ISSUE TITLE
PUT Education Endpoint School field 400 error

### ISSUE THREAD

**Issue by Clarkbaum** [sogetiprofileteam/profile-api#40]
current education model
{
id - string($uuid)
school - #/definitions/SchoolSchool{...}
levelOfDegree - string
subject - string
startDate - string($date-time)
endDate - string($date-time)
}

when hitting the PUT Education endpoint with the full consultant I get a 400 "Error converting value "institute name" to type 'sogeti_portfolio_api.Models.School'. Path 'education[0].school', line 1, position 1426."

below is what my education array looks like which is part of the consultant 
education: Array(1)
   0: endDate: "11/11/1111"
      levelOfDegree: "title"
      school: "instiute name"
      __proto__: Object
   length: 1
   __proto__: Array(0)

**Comment by danielthompson** [comment]
The problem is that the Education endpoint expects a `School` object...

https://github.com/sogetiprofileteam/profile-api/blob/master/sogeti-portfolio-api/Controllers/EducationController.cs#L11

...which only has `name` and `id`fields:

https://github.com/sogetiprofileteam/profile-api/blob/master/sogeti-portfolio-api/Models/School.cs

To fix, either add the desired fields to `School` or change the controller type to `Education`, which does have them:

https://github.com/sogetiprofileteam/profile-api/blob/master/sogeti-portfolio-api/Models/Education.cs

**Comment by TylerAlexanderTX** [comment]
Yeah I'm pretty sure the controller type needs to change back to Education instead of School. When I wrote the controller originally I had Education as the type. It looks like when it got refactored a couple weeks ago Andrew changed it to School by mistake.

**Comment by Clarkbaum** [comment]
any update on this issue? @TylerAlexanderTX @zwiseman

**Comment by Clarkbaum** [comment]
https://github.com/sogetiprofileteam/profile-api/pull/44
